doc/react/README.md

```diff
@@ -0,0 +1,146 @@
+# @hatchifyjs/react
+
+- [What is hatchifyjs/react](#what-is-hatchifyjsreact)
+  - [TypeScript](#typescript)
+  - [Types](#types)
+  - [createJsonapiClient](#createjsonapiclient)
+  - [HatchifyProvider](#hatchifyprovider)
+  - [hatchifyReact](#hatchifyreact)
+    - [Everything](#everything)
+    - [components](#components)
+    - [model](#model)
+    - [state](#state)
+      - [useDataGridState](#usedatagridstate)
+  - [MUI Components](#mui-components)
+
+## What is hatchifyjs/react?
+
+@hatchifyjs/react is a model-driven library of components for your hatchify app. By defining the schemas (AKA models) of your backend resources, @hatchifyjs/react will provide you with a set of components that you can use across your React app.
+
+### TypeScript
+
+hatchifyjs/react provides TypeScript support. Here's an example of how two schemas (`Todo` and `User`) provide auto-completion for an instantiated hatchifyjs/react app (`hatchedReact`):![react TypeScript](doc/attachments/reactTs.gif)
```

Comment:
That's a great question. I've only been doing it this way, and did not realize they could be uploaded to Github.

Reply:
**Update:** image locations are being kept as-is. There's no significant risk to having some in the repo and others uploaded via GitHub, especially since the expectation is that they'll all be unified once these docs are eventually moved to a docs site.